rust/operator-binary/src/main.rs

```diff
@@ -31,14 +32,19 @@ struct Opts {
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    stackable_operator::logging::initialize_logging("TRINO_OPERATOR_LOG");
+    stackable_operator::logging::initialize_logging(
+        "TRINO_OPERATOR_LOG",
+        APP_NAME,
+        TracingTarget::None,
```

Comment:
Please have a look here on how to handle the tracing_target
https://github.com/stackabletech/commons-operator/blob/75418fb2bb8b58ba27db25f74f5e1020427728ee/rust/operator-binary/src/main.rs#L26-L43

Reply:
oh yeah thanks done!